apps/src/templates/instructions/CollapserButton.jsx

```diff
@@ -26,7 +26,7 @@ const CollapserButton = props => (
         icon={props.collapsed ? 'chevron-circle-down' : 'chevron-circle-up'}
         style={styles.collapseIcon}
     />
-    {props.collapsed ? msg.more() : msg.hide()}
+    {props.collapsed ? msg.more() : msg.less()}
```

Comment:
`$ git grep -Ei "(msg|i18n).hide\("` returns nothing; I think we can safely get rid of it.